lib/ibmdb.js

```diff
@@ -223,10 +230,18 @@ IBMDB.prototype.executeSQL = function(sql, params, options, callback) {
     }
   }
 
-  conn.query(sql, params, function(err, data, more) {
+  // Build the stmt object that will be passed into the query call.
+  // This is done because the query call can take an object or a set
+  // of parameters.  Depending on the SQL being passed in the call with
+  // parameters may fail due to improper handling in the lower levels.
```

Comment:
Changed to "in the ibm_db module"

The reason these changes are being made is there are bugs in the ibm_db module which will eventually get fixed.  When they do they will cause this module to break without the changes I've made.